<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wavetableBrowser</class>
 <widget class="QDialog" name="wavetableBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ratio</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Select Table Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>160</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="slopePair">
            <property name="text">
             <string>Slope Pair</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="cycle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Cycle</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Numer of Waveforms:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="tableSize"/>
    </item>
    <item>
     <widget class="QLabel" name="tableSizeWarning">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>(the first N will be used)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Select Table:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="selectTable"/>
    </item>
    <item>
     <widget class="QPushButton" name="swapTable">
      <property name="text">
       <string>Swap</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>611</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="tableViz">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="WavetableVizPanel" name="viz3" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="WavetableVizPanel" name="viz1" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="WavetableVizPanel" name="viz2" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="WavetableVizPanel" name="viz4" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WavetableVizPanel</class>
   <extends>QWidget</extends>
   <header>wavetable_viz_panel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
